internal/extsvc/github/v4_test.go

```diff
@@ -367,6 +367,21 @@ func TestMarkPullRequestReadyForReview(t *testing.T) {
 	}
 }
 
+func TestCreatePullRequestComment(t *testing.T) {
+	cli, save := newV4Client(t, "CreatePullRequestComment")
+	defer save()
+
+	pr := &PullRequest{
+		// github.com/sourcegraph/automation-testing/pull/44
```

Comment:
Let's make this a full URL to make it easier to click from an editor:

```suggestion
		// https://github.com/sourcegraph/automation-testing/pull/44
```